server/utils/checkYourAnswersUtils.ts

```diff
@@ -64,7 +64,9 @@ export const addPageAnswersToItemsArray = (params: {
   const { items, application, task, pageKey, questions, outputFormat } = params
   const PageClass = getPage(task, pageKey)
 
-  const page = new PageClass({}, application)
+  const body = application?.data?.[task]?.[pageKey]
+
+  const page = new PageClass(body, application)
 
```

Comment:
is it possible to be any more specific about the scenarios where the `body` is needed?

Reply:
yes, absolutely. It's a bit of a glaring oversight that hasn't been an issue until now because on all the pages where we are relying on a `response` method (`acct`, for example), we're actually pulling the data manually from the application, rather than from the page body. But CPP details is the first exception to this. It doesn't make sense to pull the data from the application because it should be accessible in the `Page` body.

Reply:
(I'll add this to the commit message as well)